src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.tsx

```diff
@@ -30,86 +41,41 @@ const getDaysOpen = (incident) => {
   const hasDaysOpen =
     incident.status && !statusesWithoutDaysOpen.includes(incident.status.state)
   const start = hasDaysOpen && parseISO(incident.created_at)
-  return hasDaysOpen ? -differenceInCalendarDays(start, new Date()) : '-'
+  return start ? -differenceInCalendarDays(start, new Date()) : '-'
```

Comment:
If this change doesn't break the corresponding test, is the test coverage sufficient?

Reply:
We're not verifying that `created_at` is indeed a date. In my opinion we shouldn't need to validate the format of backend data - what do you think?

Reply:
We shouldn't, no. But, if we're relying on a valid date format, we should have a test for it, I guess.

Reply:
Added the test